bpf-gpl/tc.c

```diff
@@ -1272,6 +1253,82 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct __sk_buff *skb,
 	}
 }
 
+static CALI_BPF_INLINE struct fwd calico_tc_skb_icmp_accepted(struct __sk_buff *skb, struct iphdr *ip, 
```

Comment:
`_accepted` ? 

As much as I favour breaking code into small brief functions, since it is the core of `calico_tc_skb_send_icmp_replies` itself, I would perhaps put all the code straight there.